app/controllers/stats/usage_controller.rb

```diff
@@ -7,8 +7,7 @@ class Stats::UsageController < Stats::ServiceBaseController
   liquify if: :buyer_domain?
 
   def index
-    @methods = @service.method_metrics
-    @metrics = @service.metrics.top_level
+    @methods, @metrics = @service.all_metrics.partition(&:method_metric?)
```

Comment:
If there are many metrics and methods it might bloat the memory

Reply:
This is a problem we already have now.